packages/core/src/utils/markers.ts

```diff
@@ -0,0 +1,108 @@
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+export interface MarkerStyle {
+  strokeWidth?: string | number
+  stroke?: string
+  fill?: string
+}
```

Comment:
The type also exists in the types package - mind deleting this and changing the paths?

Reply:
Don't mind if I do. https://github.com/Dataport/polar/pull/42/commits/cdb594111def412933b2b2fcab5b4a1944afcd8e